packages/perseus/src/widgets/graded-group/graded-group.tsx

```diff
@@ -392,6 +395,7 @@ export class GradedGroup
                                 apiOptions={apiOptions}
                                 linterContext={this.props.linterContext}
                                 strings={this.context.strings}
+                                showSolutions={showSolutions}
```

Comment:
Does `showSolutions` this apply to the hint? Or rather, do we see rationales showing up in hint content for graded-groups?